.releaserc.json

```diff
@@ -0,0 +1,74 @@
+{
+  "branches": [
+    "master"
+  ],
+  "plugins": [
+    [
+      "@semantic-release/commit-analyzer",
+      {
+        "preset": "angular",
+        "parserOpts": {
+          "noteKeywords": [
+            "BREAKING CHANGE",
+            "BREAKING CHANGES"
+          ]
+        }
+      }
+    ],
+    [
+      "@semantic-release/release-notes-generator",
+      {
+        "preset": "angular",
+        "parserOpts": {
+          "noteKeywords": [
+            "BREAKING CHANGE",
+            "BREAKING CHANGES"
+          ]
+        },
+        "writerOpts": {
+          "commitsSort": [
+            "subject",
+            "scope"
+          ],
+          "owner": "uittorio"

```

Comment:
Can we use another username? I would like to use typescripttdd.

Reply:
@typescripttdd is a GitHub account?
If so, I will make the change.

Reply:
Yes it's a github account I've created to prevent all my stats to be changed on github. This account push every night in this repo :P

Reply:
Done.